### ISSUE TITLE
create multi database

### ISSUE THREAD

**Issue by yaming116** [Raizlabs/DBFlow#260]
I need create multi database,it has same table,how do it?  thanks.

**Comment by agrosner** [comment]
You can only associate one table with one database unfortunately. What you _can_ do is extend that table class with another `@Table` that specifies a different `databaseName` with the same `tableName()`

**Comment by yaming116** [comment]
I used sqlcipher database every times will add 3000 data, so i need create database in run time.

**Comment by agrosner** [comment]
DBFlow constructs the database code and interactions pre-compile time so you cannot create a database at runtime with this library. Sorry, I did not understand the first part of what you said, could you rephrase?

**Comment by yaming116** [comment]
I am using an encrypted database, each will be down from the server to synchronize data about 3000, at least twice a day.
When the data is greater than 10,000 inquiries and insert will slow down a lot, so you need to create multiple databases with the same.

**Comment by mkovalyk** [comment]
@yaming116 did you found any workaround how to do this?

**Comment by yaming116** [comment]
@mkovalyk no, this not support
